src/Rule/Uuid.php

```diff
@@ -23,17 +21,41 @@ class Uuid extends Regex
     const INVALID_UUID = 'Uuid::INVALID_UUID';
 
     /**
-     * UUID Version 4.
+     * UUID NIL & version binary masks
      */
-    const UUID_V4 = 4;
+    const UUID_NIL = 0b000001;
+    const UUID_V1  = 0b000010;
+    const UUID_V2  = 0b000100;
+    const UUID_V3  = 0b001000;
+    const UUID_V4  = 0b010000;
+    const UUID_V5  = 0b100000;
 
     /**
      * An array of all validation regexes.
      *
      * @var array
      */
     protected $regexes = [
-        4 => '~^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$~i',      // UUIDv4 Format
```

Comment:
Ah, I didn't notice that. But then again, it has been accepting all GUIDs we have given it so far. So I feel more comfortable with changing the default.